### ISSUE TITLE
Unable to burn dvd

### ISSUE THREAD

**Issue by grepper** [grepper/tovid#78]
_From [El.Nego](https://code.google.com/u/El.Nego/) on October 27, 2008 01:29:30_

Hey guys I know this is still not a completed feature but thought this
might help, here is the output from tovid when trying to burn, tried with
authoring and without, with burning and without, it was always the same.
Used ffmpeg for encoding

STAT: VOBU 0 at 0MB, 19 PGCS

/usr/share/tovid/makedvd: line 319:  8226 Segmentation fault      dvdauthor
-x "/tmp/Untitled_disc.xml"
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
makedvd encountered an error during the DVD creation process:
Could not create the DVD-Video disc structure in /tmp/Untitled_disc.
Leaving /tmp/Untitled_disc for inspection.
See if anything in the above output helps you diagnose the
problem, and please file a bug report at tovid.org (_not_
the dvdauthor list) containing the above output.
Sorry for the inconvenience!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

_Original issue: http://code.google.com/p/tovid/issues/detail?id=77_

**Comment by grepper** [comment]
_From [grepper@gmail.com](https://code.google.com/u/grepper@gmail.com/) on April 16, 2009 06:24:03_

This report has no version indicted and not enough debugging information to 
investigate.  It can be re-opened but would need the xml attached as well as the 
versions of the programs used (tovid, dvdauthor).  I am marking it invalid.

**Status:** Invalid

**Comment by grepper** [comment]
_From [kngh...@gmail.com](https://code.google.com/u/114874735848210842145/) on July 06, 2009 19:44:51_

I encountered the same error message.

The problem lies upon the fact that the mpeg file doesn't really fit the VOB format.  

This issue happened to me because tovid *THINK* some of my mpeg is compliant with VOB
format and obviously it didn't.  I used "-force" option to re-encode everything and
that problem went away.

So, the "bug" may be lies upon the mpeg-checking mechanism.

I am using version 0.31 on Linux
